molecule/resources/tests/test_python3.py

```diff
@@ -22,7 +22,7 @@ def test_omero_version(host):
 
 def test_postgres_version(host):
     ver = host.check_output("psql --version")
-    assert ver.startswith('psql (PostgreSQL) 13')
+    assert ver.startswith('psql (PostgreSQL) 16')
```

Comment:
better in a separate PR

Reply:
reverted back to 13 in https://github.com/ome/ansible-role-omero-server/pull/82/commits/a2b477ab2aca918af043eed3f70ae78172419c40